openmdao/core/driver.py

```diff
@@ -77,6 +77,8 @@ class Driver(object):
         Metadata pertaining to total coloring.
     _total_jac_sparsity : dict, str, or None
         Specifies sparsity of sub-jacobians of the total jacobian. Only used by pyOptSparseDriver.
+    _total_jac_format : str
+        Specifies the format of the total jacobian.
```

Comment:
should probably list the allowable values here, or point to where it's documented in `compute_totals`

Reply:
done